### PR TITLE
SIMDutils code refactoring

### PR DESCRIPTION
**Pull request by arunetm**
Refactoring SIMDUtils.

1. Clean up Js name space by moving the helper api to SIMDUtils class.
2. Re implemented the logic for Extract/ReplaceLane helpers in SIMDUtils for code size reduction and better reuse. 
3. Changed the helper call sites to use the static member functions from SIMDUtils.
4. Minor cosmetic changes.

### CODE REVIEW

**Comment by arunetm** [comment]
@abchatra please review.

**Comment by abchatra** [comment]
Please add details on other changes apart from moving the Js namespace API to SIMDUtils class?

**Comment by arunetm** [comment]
@abchatra, updated the PR description to summarize all the changes that are part of this PR.

File: lib/Runtime/Language/SimdUtils.cpp [hunk 1]
```diff
@@ -22,23 +47,20 @@ namespace Js
...
         {
             JavascriptError::ThrowRangeError(scriptContext, JSERR_SimdLaneRangeError);
         }
-
         return int32Value;
     }
 
     // Is Number with int32 value.
-    int32 SIMDCheckInt32Number(ScriptContext* scriptContext, Var value)
+    int32 SIMDUtils::SIMDCheckInt32Number(ScriptContext* scriptContext, const Var value)
```
Comment (abchatra) [inline lib/Runtime/Language/SimdUtils.cpp]: Why should this return int32? Why not uint32?

Comment (arunetm) [inline lib/Runtime/Language/SimdUtils.cpp]: Yes, these can be restricted to uint32. Changed the lane index's within helpers to be of uint32 type. Thanks.

**Comment by abchatra** [comment]
:shipit:

**Comment by arunetm** [comment]
@abchatra, Are there any additional changes that you think needed? If not, can you please help to sign the PR off? :)

**Comment by arunetm** [comment]
Updating PR after squash + rebase.

**Pull request merged**